qiita_core/test/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -----------------------------------------------------------------------------
```

Comment:
Test __init__.py should be empty, we just really want them so that our distributions ship with the test suite.